### DISCUSSION TITLE
Blog Sidebar Overview

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#767] [General]
# Blog Sidebar Overview

The blog module supports an optional sidebar layout.

https://hbstack.dev/modules/blog/sidebar/overview/

**Comment by LITUATUI** [comment]
Is it possible to use multiple custom widgets?

I would like to have in the sticky sidebar the following:

1. custom widget 1 (Boostrap carousel)
2. featured/recent posts
3. custom widget 2 (Adsense ad)

**Reply by razonyang** [reply]
yes, you can create another custom module similar to hb-custom, or put custom widget same hook and change the sequence by setting `order` CSS property.
